crates/holochain/src/conductor/conductor.rs

```diff
@@ -1871,6 +1877,30 @@ mod cell_impls {
             self.running_cells
                 .share_ref(|cells| cells.keys().cloned().collect())
         }
+
+        /// Close source chain of the given cell. Closing a source chain makes it read-only.
+        pub async fn close_chain(conductor: Arc<Self>, cell_id: CellId) -> ConductorApiResult<()> {
+            let action_builder = builder::CloseChain::new(cell_id.dna_hash().clone());
```

Comment:
This doesn't feel right. CloseChain is for pointing to a new chain during a migration. This is using that same semantic, closing this chain only to point back to itself!

It should be sufficient to delete the agent pub key for this chain.

Reply:
I interpreted the instruction in the issue that way https://github.com/holochain/holochain/issues/3595#issue-2235697450

Reply:
The issue isn't worded that well but from refinement discussions, my understanding was that a chain gets closed *either* by marking it closed to move to a new DNA or by revoking the agent key. If either of those has happened then sys validation should treat it as the end of the chain and not permit new actions.

I agree with @maackle here, it should be a delete that gets committed rather than a chain close

Reply:
Totally, had forgotten to push.